src/main/java/cinema/Main.java

```diff
@@ -13,5 +19,23 @@ public static void main(String[] args) {
         MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         movieService.add(movie);
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(50);
+        cinemaHall.setDescription("Blue");
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
+        cinemaHallService.add(cinemaHall);
+        cinemaHallService.getAll().forEach(System.out::println);
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(movie);
+        movieSession.setCinemaHall(cinemaHall);
+        movieSession.setShowTime(LocalDateTime.of(2021, 2, 1, 20, 23));
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
+        movieSessionService.add(movieSession);
+        movieSessionService.findAvailableSessions(movieSession.getId(), LocalDate.now())
```

Comment:
why do you pass the id of moviesession here?

Reply:
Fixed